c++/src/kj/exception.c++

```diff
@@ -632,6 +636,7 @@ void printStackTraceOnCrash() {
   stack.ss_sp = reinterpret_cast<char*>(mmap(
       nullptr, stack.ss_size, PROT_READ | PROT_WRITE,
       MAP_ANONYMOUS | MAP_PRIVATE | MAP_GROWSDOWN, -1, 0));
+#if !defined(__EMSCRIPTEN__)
```

Comment:
Presumably this should be moved up to the top of this function -- there's no reason to allocate the stack if you're not going to use it.

Reply:
Or maybe `printStackTracesOnCrash()` should not be available at all in emscripten mode? The terminate handler is arguably relevant but mostly this function won't do what it's intended to do.